scripts/tslint-rule-begone.js

```diff
@@ -0,0 +1,117 @@
+import { parse } from 'comment-json';
+import fs from 'fs';
+import { format } from 'prettier';
+import sh from 'shelljs';
+import * as path from 'path';
+
+/**
+ * @param {string} filePath
+ * @returns {any}
```

Comment:
If this would be the same without the return tag, might as well leave it out.

Reply:
Yeah @JoshuaKGoldberg - as a TS eslint rep I think you'd be all about the `noExplicitAny` 😉

Reply:
Have a little more faith in me here 😛 if I remove the `any`:

```ts
        format(JSON.stringify(contents, null, 4), {
            ...prettierConfig, // CommentJSONValue | undefined
         // ~~~~~~~~~~~~~~~~~
         // Spread types may only be created from object types.
            filepath: filePath,
        }),
```

I'll switch it to defaulting to `{}`.

Reply:
d8d317550d